TDTChocolate.podspec

```diff
@@ -3,16 +3,16 @@ Pod::Spec.new do |s|
   s.version  = '2.10.0'
   s.summary  = 'Library containing additions to Cocoa and Cocoa Touch'
   s.homepage = 'https://github.com/talk-to/Chocolate'
-  s.author   = 'Talk.to'
+  s.author   = 'Directi'
   s.license  = 'BSD'
   s.source   = {
     :git => 'https://github.com/talk-to/Chocolate.git',
     :tag => "#{s.version}"
   }
   s.requires_arc = true
 
-  s.ios.deployment_target = '7.0'
-  s.osx.deployment_target = '10.8'
+  s.ios.deployment_target = '8.0'
+  s.osx.deployment_target = '10.9'
```

Comment:
The enum values `NSCalendarUnitHour` and `NSCalendarUnitDay` are available only after that version.
Reference: https://github.com/talk-to/Chocolate/pull/85/commits/bdfb624367ec192770e147ebb80d6a3d04d98cbe